Plots/Animations/NCL_animate_1.py

```diff
@@ -0,0 +1,95 @@
+"""
+NCL_animate_1.py
+===============
+This script illustrates the following concepts:
+   - Creating animations
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/animate_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/animate_1_1_lg.png
+
+Please note:
+    This code will create a .gif file locally. In order to view the gif, you 
+    will need to right click on the .gif file in your local directory, then 
+    select 'open with' then select a web browser. This has been tested using 
```

Comment:
Depending on operating system and preview settings, this does not necessarily need to be opened in a web browser. I don't think we have to tell them how to open it beyond that the script will create a gif file locally.

Reply:
Just updated, let me know what you think!